src/python/dbs/apis/dbsClient.py

```diff
@@ -607,8 +607,12 @@ def insertBulkBlock(self, blockDump):
         if len(blockDump['files']) > 0:
             fileLumiList = blockDump['files'][0]['file_lumi_list']
             if len(fileLumiList) > 0:
-                if fileLumiList.get('event_count') == None:
```

Comment:
Now that we know it's a list of dictionaries, I think you could replace most of this code with something safer, e.g.:
```
flDict = next(iter(fileLumiList), {})
if flDict.get('event_count') is None:
    frst = False
```

It might be better to rename `frst` to `hasEventCount`, or something like that as well.